projects/common/src/utilities/formatters/date/date-formatter.ts

```diff
@@ -54,7 +54,7 @@ export class DateFormatter {
         return 'd MMM y h:mm a';
       case DateFormatMode.DateAndTimeWithSeconds:
       default:
-        return 'y-M-d hh:mm:ss a';
+        return 'y-MM-dd hh:mm:ss a';
```

Comment:
Oh thank god.

Is the cell renderer using this so it will be fixed as well?

Reply:
sure is

Reply:
sure is.

Reply:
sure is..